_members/mike-lippincott.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Mike Lippincott
+image: images/team/\
+description: PhD Student
+role: phd
+aliases:
+    - Mike Lippincott
+    - Michael Lippincott
+    - Mike
+    - Michael J. Lippincott
+    - lippincm
+links:
+  orcid: 0000-0002-8637-1448
+  home-page: https://github.com/MikeLippincott
+  email: michael.lippincott@cuanschutz.com
+  github: MikeLippincott
+  twitter: mike_lippincott
+---
+
+Mike Lippincott is a Cell Biology, Stem Cells, and Development (CSD) PhD student at the University of Colorado, Anschutz Medical Campus. Mike joined the Way Lab in July 2023 and is looking to understand how cellular morphologies change in the context of cellular stress and how organelle interactions affect cellular states.
+
+Mike earned a B.S. in Biochemistry and Data Science from [Maryville University of Saint Louis](https://www.maryville.edu/) in 2021. Here he conducted research in biochemical pathway manipulations in sugar cane, CRISPR gene editing in non-mendelian genetic plant models, plasma based atomic emission spectroscopy, and cellular transcriptomic investigation of telomerase activating drug TA-65.
+
+After graduating from [Maryville](https://www.maryville.edu/), Mike did his post-baccalaureate training at the [University of North Caroina at Chapel Hill](https://www.unc.edu/) where he developed software to analyze telomeric sequence homology in [Caenorabditis elegans](http://www.wormbook.org/) and Humans. 
+
+Mike is passionate about collaborative team science and bridging the gap between scientific fields to promote interdisciplinary science. 
```

Comment:
love it! Can you add your _why_ statement? Why are you passionate? (it currently reads as a _what_ statement, which is less powerful)